spec.html

```diff
@@ -42032,7 +42032,7 @@ <h1>Atomics.sub ( _typedArray_, _index_, _value_ )</h1>
 
     <emu-clause id="sec-atomics.wait">
       <h1>Atomics.wait ( _typedArray_, _index_, _value_, _timeout_ )</h1>
-      <p>`Atomics.wait` puts the calling agent in a wait queue and puts it to sleep until it is notified or the sleep times out. The following steps are taken:</p>
+      <p>Puts the calling agent in a wait queue and suspends it until notified or the wait times out, returning a String differentiating those cases. The following steps are taken:</p>
```

Comment:
```suggestion
      <p>`Atomics.wait` puts the calling agent in a wait queue and suspends it until notified or the wait times out, returning a String differentiating those cases. The following steps are taken:</p>
```
We don't normally use sentence fragments in this position, I don't think? We're inconsistent in the terminology here, but I don't think we have that particular inconsistency, at least not often.

Also I prefer "until it is notified" rather than "until notified", since it makes explicit which thing is being notified.

Reply:
Sentence fragments here are not <em>un</em>common (there are lots of examples like `<p>Returns …` and `<p>Performs …` and `<p>Produces …`, cf. [Number.prototype.toLocaleString](https://tc39.es/ecma262/multipage/numbers-and-dates.html#sec-number.prototype.tolocalestring)/[BigInt.prototype.toLocaleString](https://tc39.es/ecma262/multipage/numbers-and-dates.html#sec-bigint.prototype.tolocalestring) and [Math](https://tc39.es/ecma262/multipage/numbers-and-dates.html#sec-function-properties-of-the-math-object) and [String.prototype.matchAll](https://tc39.es/ecma262/multipage/text-processing.html#sec-string.prototype.matchall) and [String.prototype.split](https://tc39.es/ecma262/multipage/text-processing.html#sec-string.prototype.split) and [RegExp.prototype.exec ( string )](https://tc39.es/ecma262/multipage/text-processing.html#sec-regexp.prototype.exec) and [Array.prototype.concat](https://tc39.es/ecma262/multipage/indexed-collections.html#sec-array.prototype.concat) and [%TypedArray%.prototype.subarray](https://tc39.es/ecma262/multipage/indexed-collections.html#sec-%typedarray%.prototype.subarray), with honorable mentions for [Bitwise Shift Operators](https://tc39.es/ecma262/multipage/ecmascript-language-expressions.html#sec-bitwise-shift-operators) and [String.prototype.charCodeAt](https://tc39.es/ecma262/multipage/text-processing.html#sec-string.prototype.charcodeat)/[String.prototype.codePointAt](https://tc39.es/ecma262/multipage/text-processing.html#sec-string.prototype.codepointat) [in which it is part of a note]).

But I actually share your grammatical preference, and don't remember my motivation here so I have accepted your suggestions. And in the long run, I expect this to be resolved by introducing more structure so the phrasing will become almost moot.

Reply:
> Sentence fragments here are not _un_common (there are lots of examples like `<p>Returns …` and `<p>Performs …` and `<p>Produces …`, 

(Commit c6ca25fc of PR #2592 changes these by inserting "This function" or "This method".)

Reply:
Thanks @jmdyck, I fully support the efforts of #2592. @bakkot Should I pull forward and merge its changes here?
```suggestion
      <p>This function puts the calling agent in a wait queue and suspends it until it is notified or the wait times out, returning a String differentiating those cases.</p>
      <p>It performs the following steps when called:</p>
```

Reply:
> Should I pull forward and merge its changes here?

Ehhh that seems pretty unrelated. I'd prefer not to try to tweak wording piecemeal; better for each PR to do its own thing.